ui/src/app/components/tables.tsx

```diff
@@ -1,125 +1,13 @@
 import colors from 'app/styles/colors';
-import {nextSort} from 'app/utils';
-import * as fp from 'lodash/fp';
 import * as React from 'react';
-import Pagination from 'react-paginating';
-import {ClrIcon} from './icons';
-
-const rowHeight = '1.5rem';
 
 const styles = {
-  flexCell: ({basis = 0, grow = 1, shrink = 1, min: minWidth = 0, max} = {} as any) => ({
-    flexGrow: grow,
-    flexShrink: shrink,
-    flexBasis: basis,
-    minWidth,
-    maxWidth: max
-  }),
-  headerRow: {
-    display: 'flex', height: rowHeight,
-    borderBottom: '1px solid #ccc', backgroundColor: '#fafafa'
-  },
-  bodyRow: {
-    display: 'flex', height: rowHeight,
-    borderTop: '1px solid #ddd', backgroundColor: 'white'
-  },
-  cell: {
-    display: 'flex', alignItems: 'center',
-    paddingLeft: '0.5rem', paddingRight: '0.5rem'
-  },
-  paginationRow: {
-    display: 'flex', alignItems: 'center', height: rowHeight,
-    border: '1px solid #ccc', backgroundColor: '#fafafa', fontSize: 13
-  },
   paddedTable: {
     color: colors.primary, fontSize: 13, lineHeight: '24px',
     background: colors.white, padding: '0.2rem 0.5rem', display: 'flex'
   }
 };
 
-export const SimpleTable = ({columns, rowCount, rowProps}) => {
```

Comment:
How did you identify this dead code? IDE? Script?

Reply:
IDE, and then find all in directory on UI showed no uses.